zulipterminal/model.py

```diff
@@ -480,9 +481,8 @@ def update_stream_message(self, topic: str, message_id: int,
 
         return response['result'] == 'success'
 
-    def generate_all_emoji_data(self,
-                                custom_emoji: Dict[str,
-                                                   Any]) -> NamedEmojiData:
+    def generate_all_emoji_data(self, custom_emoji: Dict[str,
+                                RealmEmojiData]) -> NamedEmojiData:
```

Comment:
You don't need to specify the refactor location - this is about refining the typing information from the server.